wsd/Storage.cpp

```diff
@@ -651,6 +652,8 @@ std::unique_ptr<WopiStorage::WOPIFileInfo> WopiStorage::getWOPIFileInfo(const Au
                                                                         const std::string& cookies,
                                                                         LockContext& lockCtx)
 {
+    ProfileZone profileZone("getWOPIFileInfo", { {"url", _fileUrl} });
```

Comment:
Lovely =) Ash'll need to convert it to an async pair as/when we get async save in I guess but that's fine.